packages/react/src/card/card.tsx

```diff
@@ -204,16 +204,27 @@ export interface CardProps extends CardBaseProps {
    * @default "lighter-brand"
    * */
   color?: "white" | "lighter-brand" | "light-grey-fill";
+  /**
+   * On cards that can have images to the left or right of the text, (fullwidth or focus),
+   * left position is default. Setting imageRight to true, puts the image to the right.
+   *
+   * @default false
+   * */
+  imageRight?: boolean;
 }
 
 interface CardFocusProps extends CardBaseProps {
   as?: "section" | "div" | "article" | "aside";
   variant: "focus";
   color: "darker";
+  imageRight?: boolean;
```

Comment:
Looks to me this is only applicable to `variant=full-width`.

Then I suggest we make `interface CardFullWidthProps extends CardBaseProps`, same as this one
and then do

```tsx
export const Card = forwardRef<HTMLDivElement, CardProps | CardFocusProps | CardFullWidthProps>(
```
down below